### PR TITLE
Add Datadog RUM

### PR DESCRIPTION
**Pull request by we-ai**
This PR is to add Datadog RUM to biospecimen app for evaluation, as described in issue [#613](https://github.com/episphere/connect/issues/613).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,12 +50,18 @@ window.onload = () => {
...
+        window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
     }
     else if(location.host === urls.stage) {
         !firebase.apps.length ? firebase.initializeApp(stageFirebaseConfig()) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init({ ...DatadogDevConfig, env: 'stage' });
+        window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
     }
     else {
         !firebase.apps.length ? firebase.initializeApp(devFirebaseConfig()) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init(DatadogDevConfig);
```
Comment (anthonypetersen) [inline index.js:63]: Just a suggestion from personal preference... Declare the Datadog variable without an "env" parameter and don't name it "dev config", then in code always pass which environment you will be using. Might help clarify that the config is truly for all tiers.

File: index.js [hunk 2]
```diff
@@ -36,12 +50,18 @@ window.onload = () => {
...
+        window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
```
Comment (anthonypetersen) [inline index.js:64]: Could we pull this line outside of this IF-ELSE block and just declare it once?

**Review by anthonypetersen — APPROVED** [review]
Looks good. Added comments that are simply suggestions / preferences, I don't feel too strongly about it but whatever we decide we should do the same thing on the other applications.

File: index.js (hunk 1 above)
Comment (we-ai) [reply index.js]: Updated.

File: index.js (hunk 2 above)
Comment (we-ai) [reply index.js]: Updated.

**Comment by anthonypetersen** [comment]
Looks great :) @we-ai

**Pull request merged**